src/dex-ui-components/base/Icons/TransactionIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import { createIcon } from "@chakra-ui/react";
+
+export const TransactionIcon = createIcon({
+  displayName: "TransactionIcon",
+  viewBox: "0 0 22 22",
+  path: [
+    <path
+      d="M9 8L2 15M2 15L7.6 15M2 15L2 9.4"
```

Comment:
we could pas attribute `curentColor` to fill and pass the color from the component so that it can take all chakra styling objects including `color` object.

Reply:
Replaced static values for colour in icons with variable currentColor.